web/src/providers/TestDefinition/hooks/useDraftMode.ts

```diff
@@ -1,8 +1,6 @@
-import {useEffect, useState} from 'react';
-
-const useDraftMode = (isDefaultDraftMode = false) => {
-  const [isDraftMode, setIsDraftMode] = useState(isDefaultDraftMode);
+import {useEffect} from 'react';
 
+const useDraftMode = (isDraftMode: boolean) => {
```

Comment:
Not for this PR. but we could refactor this into a generic hook.

Reply:
Let me move it to the `hooks` folder and make the proper adjustments